decidim-admin/app/packs/stylesheets/decidim/admin/_datepicker.scss

```diff
@@ -0,0 +1,62 @@
+.wc-datepicker {
+  @apply z-[2];
+
+  &__current-month {
+    @apply mt-4;
+  }
+
+  &__next-month-button,
+  &__previous-month-button {
+    > svg {
+      @apply fill-none w-16 h-16;
+    }
+  }
+}
+
+.datepicker {
+  &_clock-button {
```

Comment:
According to BEM the separator between the block and the element should be two underscores (`__`) instead of only one (`_`). You need to change it also in the rest of the classes of this same file too. 
```suggestion
  &__clock-button {
```